lib/init/src/root.zig

```diff
@@ -4,10 +4,15 @@
 const std = @import("std");
 const testing = std.testing;
 
+/// Returns the sum of `a` and `b`.
 pub export fn add(a: i32, b: i32) i32 {
     return a + b;
 }
 
+test add {
+    try testing.expect(add(3, 7) == 10);
+}
```

Comment:
What's the point of having this test twice? +1 for making this a doctest but duplicating it seems silly.